deploy/route-monitor-operator-manager-role.ClusterRole.yaml

```diff
@@ -20,6 +20,7 @@ rules:
     resources:
       - services
       - configmaps
+      - secrets
```

Comment:
Same comment around create/delete secrets. This needs to be scoped differently.

Reply:
+1, seperated this out with get, list and watch permissions only for resource secrets.